generators/server/templates/_pom.xml

```diff
@@ -118,7 +118,7 @@
         <sonar.tests>${project.basedir}/<%= TEST_DIR %></sonar.tests>
     </properties>
 
-    <%_ if (applicationType == 'microservice' || applicationType == 'gateway') { _%>
+    <%_ if (applicationType === 'microservice' || applicationType === 'gateway') { _%>
     <dependencyManagement>
         <dependencies>
```

Comment:
We'll also need dependency management for spring-cloud

    <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>org.springframework.cloud</groupId>
                <artifactId>spring-cloud-dependencies</artifactId>
                <version>${spring-cloud.version}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
        </dependencies>
    </dependencyManagement>